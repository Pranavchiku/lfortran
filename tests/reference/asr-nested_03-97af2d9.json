{
    "basename": "asr-nested_03-97af2d9",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/nested_03.f90",
    "infile_hash": "f5d522eba458efa50018d8ebcd4df6b9098d3fd59164c530fdc522e5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-nested_03-97af2d9.stdout",
    "stdout_hash": "5360cf7fdf4ce02ab5ed8a07e26f3ff678f66581c97814ec55d76b9d",
    "stderr": "asr-nested_03-97af2d9.stderr",
    "stderr_hash": "73aa277db1830925b9a6931d2b2dfff18bb26cb13906197d587c52f3",
    "returncode": 0
}