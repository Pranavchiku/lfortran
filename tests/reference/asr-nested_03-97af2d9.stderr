warning: Implied save feature
 --> tests/../integration_tests/nested_03.f90:7:1
  |
7 | real :: x = 6
  | ^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
